packages/trader/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -0,0 +1,202 @@
+                                </div>
+                                <p className='personal-details-form__description'>
+                                    <Localize
+                                        i18n_default_text={'Any information you provide is confidential and will be used for verification purposes only.'}
+                                    />
+                                </p>
+                            </div>
+                            <FormSubmitButton
+                                is_disabled={
+                                    // eslint-disable-next-line no-unused-vars
+                                    isSubmitting ||
+                                    Object.keys(errors).length > 0
+                                }
+                                label='Next'
+                                has_cancel
+                                cancel_label='Previous'
+                                onCancel={this.props.onCancel}
+                            />
+                        </form>
+                    )
+                }
+            </Formik>
+        );
+    }
+
+    validateAddressDetails = (values) => {
+        const validations = {
+            address_line_1: [
+                v => !!v,
+                v => /^[\p{L}\p{Nd}\s'.,:;()@#/-]{1,70}$/gu.exec(v) !== null,
+            ],
+            address_line_2: [
+                v => !v || (/^[\p{L}\p{Nd}\s'.,:;()@#/-]{0,70}$/gu.exec(v) !== null),
+            ],
+            address_city: [
+                v => !!v,
+                v => /^[\p{L}\s'.-]{1,35}$/gu.exec(v) !== null,
+            ],
+            address_state: [
+                v => /^[\p{L}\p{Nd}\s'.,-]{0,35}$/gu.exec(v) !== null,
+            ],
+            address_postcode: [
+                v => !!v,
+                v => /^[^+]{0,20}$/gu.exec(v) !== null,
+            ],
+        };
+
+        const mappedKey = {
+            address_line_1  : localize('First line of address'),
+            address_line_2  : localize('Second line of address'),
+            address_city    : localize('Town/City'),
+            address_state   : localize('State/Province'),
+            address_postcode: localize('Postal/ZIP Code'),
+        };
+
+        const required_messages = [
+            '%s is required',
+            '%s is not in a proper format.',
+        ];
+
+        const optional_messages = [
+            '%s is not in a proper format.',
+        ];
+
+        const errors = {};
+
+        Object.entries(validations)
+            .forEach(([key, rules]) => {
+                const error_index = rules.findIndex(v => !v(values[key]));
+                if (error_index !== -1) {
+                    switch (key) {
+                        case 'address_state':
+                        case 'address_line_2':
+                            errors[key] =
+                                localize(optional_messages[error_index].replace('%s', mappedKey[key]));
```

Comment:
I don't think we can pass variables to localize, only passing strings gets picked up by the extraction script